.babelrc

```diff
@@ -0,0 +1,3 @@
+{
+  "presets": ["es2015"]
+}
```

Comment:
But `babel-it` is not used here. I can see this package uses `dist-it` and the latest uses `babel-cli` directly. So without this file uglification fails for me.